scripts/jenkins/workers/roles/jenkins_worker/tasks/configure_git.yml

```diff
@@ -0,0 +1,8 @@
+- name: Configure git for jenkins user
```

Comment:
You could have done this better with:
```
- git_config:
    name: name
    scope: global
    value: SUSE Jenkins Gerrit

- git_config:
    name: email
    scope: global
    value: cloud-jenkins-gerrit-owner@suse.de
```

Reply:
https://github.com/SUSE-Cloud/automation/pull/3524